app/scripts/importUiMetaData/README.md

```diff
@@ -40,19 +40,21 @@ This field specifies the correct input component for the given json path.
-| STRING_ENUM            | strings with predefined values to select from<br/> the values are defined in the schema file                                               |
-| STRING_WITH_OPTIONS    | strings with predefined values to select from, still allowing other user input<br/> the can be added via the `options` property, see below |
-| STRING_MULTI_LINE      | strings that are potentially larger                                                                                                        |
-| STRING_PRODUCT_ID      | strings referencing a product ID<br/> allows selecting from matching product IDs in the frontend                                           |
-| STRING_GROUP_ID        | strings referencing a group ID<br/> allows selecting from matching group IDs in the frontend                                               |
-| OBJECT_CWE             | special type for CWE objects ensuring that ID and name match                                                                               |
-| OBJECT_CVSS_2          | special CVSS v2 object                                                                                                                     |
-| OBJECT_CVSS_3          | special CVSS v3.0 / v3.1object                                                                                                             |
-| ARRAY_REVISION_HISTORY | special type of array, disabling some input values when the revision history is managed in the backend                                     |
+| uiType                    | type of input values                                                                                                                       |
+| :------------------------ |:-------------------------------------------------------------------------------------------------------------------------------------------|
+| STRING_DATETIME           | datetime values                                                                                                                            |
+| STRING_URI                | strings matching the URI pattern                                                                                                           |
+| STRING_ENUM               | strings with predefined values to select from<br/> the values are defined in the schema file                                               |
+| STRING_WITH_OPTIONS       | strings with predefined values to select from, still allowing other user input<br/> the can be added via the `options` property, see below |
+| STRING_MULTI_LINE         | strings that are potentially larger                                                                                                        |
+| WITH_GENERATED_PRODUCT_ID | objects with a unique `product_id` prefilled                                                                                               |
```

Comment:
I guess that one does not longer exist...

Reply:
True, I updated the readme.